static/src/javascripts/projects/commercial/modules/comscore.spec.js

```diff
@@ -34,6 +64,35 @@ describe('comscore init', () => {
 
         expect(onConsentChange).toBeCalled();
     });
+
+    describe.skip('Framework consent: running on consent', () => {
+        beforeEach(() => {
+            jest.resetAllMocks();
+        });
+
+        it('TCFv2 with consent: runs', () => {
+            onConsentChange.mockImplementation(tcfv2WithConsentMock);
+            init();
+            expect(spyConsent).toBeCalledWith(true);
```

Comment:
`spyConsent` is never called…

Reply:
what happens if you move the spy inside the test?

Reply:
Creating a `_.resetInit` did the trick, thanks @ioanna0 !